ephios/core/models/users.py

```diff
@@ -154,20 +155,22 @@ def get_workhour_items(self):
         from ephios.core.models import AbstractParticipation
 
         participations = (
-            self.participations.filter(state=AbstractParticipation.States.CONFIRMED)
+            self.participations.filter(state=AbstractParticipation.States.CONFIRMED, finished=True)
```

Comment:
I think this is not the right filter. Should be events that ended, while `finished` means ephios has processed triggers for this shift being finished.